project.clj

```diff
@@ -3,12 +3,37 @@
   :url "https://github.com/fredericksgary/test.chuck"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "0.0-3308"]
```

Comment:
although now that I think about it, cljs should probably be a dev dependency. Else anybody using test.chuck will now have cljs+etc on the classpath, which is complicated and sometimes bad.